src/borg/repository.py

```diff
@@ -843,6 +850,8 @@ def report_error(msg):
             transaction_id = self.get_transaction_id()
             current_index = self.open_index(transaction_id)
             logger.debug('Read committed index of transaction %d', transaction_id)
+        except self.AtticRepository as exc:
+            raise
```

Comment:
yes, just remove the "as exc". the plain "raise" re-raises the existing exception.